index.js

```diff
@@ -0,0 +1,70 @@
+const button = document.getElementById("button")
+const taskList = document.getElementById("taskList")
+let progressList = document.getElementsByName('process')
+let newTask = document.getElementById("newTask")
+let inicialTableTag = 
+'<tr>' +
+    '<th>' + "ID" + '</th>' +
+    '<th>' + "コメント" + '</th>' +
+    '<th>' + "状態" + '</th>' +
+'<tr>'
+taskList.innerHTML = inicialTableTag
+const tasks = [];
+
+//新規タスクの追加ボタンが押下された時の処理
+button.addEventListener("click", ()=>{
+    if(newTask.value == ""){
+        alert("入力が空です。")
+    }else{
+        //selectedProgress = checkProgress()
+        //tasks.push({ taskName: newTask.value, progress: selectedProgress });
+        tasks.push({ taskName: newTask.value, progress: "" });
+        newTask.value == ""
```

Comment:
比較ではなく、代入をされたい…のですよね？

```suggestion
        newTask.value = ""
```

Reply:
代入がしたかったです。ご指摘ありがとうございます。